src/examples/design/icons/IconShowcase.tsx

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import { Grid, Row, Col } from '@zendeskgarden/react-grid';
+import { PALETTE } from '@zendeskgarden/react-theming';
+import { ReactComponent as LeafIcon } from '@zendeskgarden/svg-icons/src/16/leaf-stroke.svg';
+import { ReactComponent as LightBulbIcon } from '@zendeskgarden/svg-icons/src/16/lightbulb-stroke.svg';
+import { ReactComponent as LightningBoltIcon } from '@zendeskgarden/svg-icons/src/16/lightning-bolt-stroke.svg';
+import { ReactComponent as LocationIcon } from '@zendeskgarden/svg-icons/src/16/location-stroke.svg';
+import { ReactComponent as SmileyIcon } from '@zendeskgarden/svg-icons/src/16/smiley-stroke.svg';
+
+const ICON_FILL = PALETTE.grey[600];
```

Comment:
Should this leverage the `theme` value through styled-components? Consumers of the design system should not access a color by pulling directly from `PALETTE`.

Reply:
I should be using `DEFAULT_THEME.palette.grey[600]` instead?

Reply:
@exelarios we'll tackle this together offline along with the grid re-work.

Reply:
No because that would still directly use the `PALETTE` object. Could use `const theme = useTheme();`. This isn't a request for change. I was more so curious if `theme` should be leveraged where possible in examples. Looks like it doesn't have to be: https://garden.zendesk.com/components/avatar#shape

Gonna resolve this comment 👍🏽 .